src/pages/docs/support/support-video-capture.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 7-29-2025
+modDate: 7-29-2025
+title: Support screen capture
+description: How to screen capture a video file to help the Octopus team resolve issues.
+navOrder: 1
+---
+
+When you contact Octopus Deploy support with an issue, we'll sometimes ask you to use our screen capture app to display the issue, so that we can understand what went wrong. This page explains how to capture this information.
+
+If you have a mic and want to narrate the issue during the screen capture, you can give the application permission to capture audio.
+
+Details on the app and the data usage:
+
+1. Recordings sent to Octopus will be used for diagnostic purposes only and will not be shared externally from Octopus in any way.
+2. All recordings are sent to an encrypted and secure Azure blob container.
```

Comment:
Do we want to be this descriptive here? Would `encrypted and secure file storage` be less of an information disclosure?